### PR TITLE
Docs ref glossary fundamental 0

### PR DESCRIPTION
**Pull request by locmai**
Update fundamental glossaries 

Signed-off-by: Loc Mai locmai0201@gmail.com

### CODE REVIEW

File: content/vi/docs/reference/glossary/annotation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Annotation
```
Comment (sftim) [inline content/vi/docs/reference/glossary/annotation.md]: I'd expect this to be localized, possibly with the original term shown as well. If this is the normal term that a Vietnamese person would use in technical writing, it's of course fine to use that.

File: content/vi/docs/reference/glossary/applications.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Applications
```
Comment (sftim) [inline content/vi/docs/reference/glossary/applications.md]: I'd expect to see this name translated / localized.

File: content/vi/docs/reference/glossary/control-plane.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Control Plane
```
Comment (sftim) [inline content/vi/docs/reference/glossary/control-plane.md]: I'd expect to see this name translated / localized.

**Comment by truongnh1992** [comment]
Hi @locmai Please resolve all the defects detected by @sftim. Thanks!

**Comment by sftim** [comment]
@locmai, I'm afraid you'd also need to rebase this PR.

**Comment by locmai** [comment]
> @locmai here's some feedback.
> 
> Please bear in mind: I don't speak any Vietnamese. I'm commenting based on how this looks to me as an English speaker. So, I might be making wrong judgements.

Hi guys, sorry for the delayed, I'm gonna update this PR soon in the next 1-2 days :D

**Comment by locmai** [comment]
Hi @sftim @truongnh1992 , I have fixed the issues above. 

Some terms could be simply translated like Annotation => "Chú thích", but the usage of the word might be different, so I left it as Tim suggested - "Chú thích (Annotation) for Label, Controller, Annotation, Selector.
For terms like Service, Pod, Deployment, etc (mostly k8s resources), we might want to keep the original terms.  

Thanks for reviewing the PR guys :D please help me double-check again!

**Pull request merged**